libopenage/renderer/opengl/texturearray.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015-2017 the openage authors. See copying.md for legal info.
+
+#pragma once
+
+#include "../resources/texture_data.h"
+#include "simple_object.h"
+
+
+namespace openage {
+namespace renderer {
+namespace opengl {
+
+/// A handle to an OpenGL texture.
+class GlTextureArray final : public GlSimpleObject {
+public:
+	/// Constructs a texture and fills it with the given data.
+	GlTextureArray(const resources::TextureData&);
```

Comment:
Should the parameter not be a `vector<TextureData>` instead? Since there's multiple of them.

Reply:
I have since written a better version of this. which includes a different constructor and a better way to send in TextureData with the submit_texture() method. Will commit that soon.